SU-HSDO/suhumsci
# SHS-5621: CSS Code Cleanup: Misc. fixes and improvements

# READY FOR REVIEW

## Summary
Fixes and improvements on misc mixins

## Need Review By (Date)
06/16

## Urgency
medium

## Steps to Test
1. Visit different pages on Traditional and Colorful, or run VRT
2. Confirm buttons still look the same. Check also the ones when there's a dark pattern.
3. Check icons still look the same
4. Check images grid still look the same
5. Check ordered and unordered list still look the same
6. Check quote wrappers still looks the same

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
